src/services/rest/http-client.js

```diff
@@ -3,6 +3,8 @@ import url from 'url';
 // see getDefaultHTTPModule for additional imports due to webpack
 
 const IS_NODE = typeof window === 'undefined';
+const HTTP_200_OK_RANGE_START = 200;
```

Comment:
Yes, but that doesn't mean _we_ (the consumers) can do anything with a 204 No Content response. Or?